packages/babel-node/src/babel-node.js

```diff
@@ -84,8 +84,12 @@ getV8Flags(async function (err, v8Flags) {
       throw err;
     }
 
+    // passthrough IPC only if babel-node itself has an IPC channel
+    const shouldPassthroughIPC = process.send != null;
```

Comment:
nit: Per [docs](https://nodejs.org/api/process.html#process_process_send_message_sendhandle_options_callback) we just need to check
```suggestion
    const shouldPassthroughIPC = process.send !== undefined;
```